Documentation/api.md

```diff
@@ -316,23 +328,32 @@ curl 'http://127.0.0.1:2379/v2/keys/foo' -vv
 < HTTP/1.1 200 OK
 < Content-Type: application/json
 < X-Etcd-Cluster-Id: 7e27652122e8b2ae
-< X-Etcd-Index: 2002
+< X-Etcd-Index: 2007
 < X-Raft-Index: 2615
 < X-Raft-Term: 2
 < Date: Mon, 05 Jan 2015 18:54:43 GMT
 < Transfer-Encoding: chunked
 < 
-{"action":"get","node":{"key":"/foo","value":"","modifiedIndex":2002,"createdIndex":2002}}
+{"action":"get","node":{"key":"/foo","value":"bar","modifiedIndex":7,"createdIndex":7}}
 ```
 
-The `X-Etcd-Index` is important. It is the index when we got the value of `/foo`.
-So we can watch again from the (`X-Etcd-Index` + 1) without missing an event after the last get.
+Unlike watches we use the `X-Etcd-Index + 1` of the response as a `waitIndex`
+instead of the node's `modifiedIndex + 1` for two reasons:
+
+1. The `X-Etcd-Index` will be greater than or equal to the `modifiedIndex`
```

Comment:
This statement is not accurate? I see watches return with X-Etcd-Index less than modifiedIndex. I set the next watch to greater of the two indicies + 1. This is exactly the question I was asking in #3007